### DISCUSSION TITLE
How can we stop tracking in iOS

### DISCUSSION THREAD

**Discussion by shinnguyen5589** [klaviyo/klaviyo-swift-sdk#87] [Archive]
Hi team,

I stop tracking in Android with the command:
// Stop tracking Kermit 
Klaviyo.resetProfile()
But in iOS, I don't find any document about this, can you update for IOS?
Thanks.

**Comment by ndurell** [comment]
We actually added that recently! However you need to potentially migrate to the new API. 
You can simply call this [method](https://github.com/klaviyo/klaviyo-swift-sdk/blob/b188ab5e4e841d24fa2765583ef0d7d75f9e6420/Sources/KlaviyoSwift/Klaviyo.swift#L412).

**Comment by ndurell** [comment]
I haven't heard much back so I'm going to close this out for now.
